docs/src/app/(public)/(content)/react/components/menubar/demos/hero/tailwind/index.tsx

```diff
@@ -0,0 +1,196 @@
+                      >
+                        Single Page
+                      </Menu.Item>
+                      <Menu.Item
+                        onClick={handleClick}
+                        className="flex cursor-default items-center justify-between gap-4 px-4 py-2 text-sm leading-4 outline-none select-none data-[highlighted]:relative data-[highlighted]:z-0 data-[highlighted]:text-gray-50 data-[highlighted]:before:absolute data-[highlighted]:before:inset-x-1 data-[highlighted]:before:inset-y-0 data-[highlighted]:before:z-[-1] data-[highlighted]:before:rounded-sm data-[highlighted]:before:bg-gray-900"
+                      >
+                        Two Pages
+                      </Menu.Item>
+                      <Menu.Item
+                        onClick={handleClick}
+                        className="flex cursor-default items-center justify-between gap-4 px-4 py-2 text-sm leading-4 outline-none select-none data-[highlighted]:relative data-[highlighted]:z-0 data-[highlighted]:text-gray-50 data-[highlighted]:before:absolute data-[highlighted]:before:inset-x-1 data-[highlighted]:before:inset-y-0 data-[highlighted]:before:z-[-1] data-[highlighted]:before:rounded-sm data-[highlighted]:before:bg-gray-900"
+                      >
+                        Continuous
+                      </Menu.Item>
+                    </Menu.Popup>
+                  </Menu.Positioner>
+                </Menu.Portal>
+              </Menu.Root>
+
+              <Menu.Separator className="mx-4 my-1.5 h-px bg-gray-200" />
+              <Menu.Item
+                onClick={handleClick}
+                className="flex cursor-default items-center justify-between gap-4 px-4 py-2 text-sm leading-4 outline-none select-none data-[highlighted]:relative data-[highlighted]:z-0 data-[highlighted]:text-gray-50 data-[highlighted]:before:absolute data-[highlighted]:before:inset-x-1 data-[highlighted]:before:inset-y-0 data-[highlighted]:before:z-[-1] data-[highlighted]:before:rounded-sm data-[highlighted]:before:bg-gray-900"
+              >
+                Full Screen
+              </Menu.Item>
+            </Menu.Popup>
+          </Menu.Positioner>
+        </Menu.Portal>
+      </Menu.Root>
+
+      <Menu.Root disabled>
+        <Menu.Trigger className="h-8 rounded px-3 text-sm font-medium text-gray-600 outline-none select-none focus-visible:bg-gray-100 data-[disabled]:opacity-50 data-[popup-open]:bg-gray-100">
+          Help
+        </Menu.Trigger>
+      </Menu.Root>
+    </Menubar>
+  );
+}
+
+function handleClick(event: React.MouseEvent) {
+  // eslint-disable-next-line no-console
+  console.log(`${(event.currentTarget as HTMLElement).innerText} clicked`);
+}
```

Comment:
Type the `MouseEvent` with `<HTMLElement> and use `textContent` (`innerText` is not the standard property)